app/src/main/java/ua/pp/marchik/root/myapplication/MainActivity.java

```diff
@@ -28,6 +28,7 @@ public void onClick(View view) {
         });
     }
 
+
     @Override
     public boolean onCreateOptionsMenu(Menu menu) {
         // Inflate the menu; this adds items to the action bar if it is present.
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Make this line start at column 5. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://77.121.149.190:9000/coding_rules#rule_key=squid%3AIndentationCheck)